src/alt/index.js

```diff
@@ -77,6 +42,51 @@ class Alt {
     return makeAction(this, 'global', name, implementation, obj)
   }
 
+  createUnsavedStore(StoreModel, ...args) {
+    StoreModel.config = fn.assign({
+      invisible: true
+    }, StoreModel.config)
+    return this.createStore(StoreModel, null, ...args)
+  }
+
+  createStore(StoreModel, iden, ...args) {
+    // pull all prototype methods out
+    const proto = utils.getInternalMethods(StoreModel, true)
+
+    // force inheritance on store
+    utils.inherit(StoreModel, Store, Object.keys(proto).reduce((obj, name) => {
```

Comment:
Nah it doesn't it should walk up the proto chain.

I doubt this will work with proper ES6 classes anyway, I may just nuke this. This was just a fun experiment :)